tools/packaging/build/agent-enclave-bundle/sgx-mode-config.filter

```diff
@@ -10,12 +10,49 @@ else
 	"minor": 125,
 	"fileMode": 438
 }] |
+.linux.devices |= . +
+[{
+	"path": "/dev/sgx_provision",
+	"type": "c",
+	"major": 10,
+	"minor": 126,
+	"fileMode": 438
+}] |
 .linux.resources.devices |= . +
 [{
 	"allow": true,
 	"type": "c",
 	"major": 10,
 	"minor": 125,
 	"access": "rwm"
+}] |
+.linux.resources.devices |= . +
+[{
+	"allow": true,
+	"type": "c",
+	"major": 10,
+	"minor": 126,
+	"access": "rwm"
+}] |
+.mounts |= . +
+[{
+	"destination": "/etc/sgx_default_qcnl.conf",
+	"type": "bind",
+	"source": "/etc/sgx_default_qcnl.conf",
```

Comment:
how the PCCS URL gets configured? This file is installed by the DCAP libs but it also needs to be configured. Or are you running a `localhost` PCCS in CI?